pages/glossary/web-page-html.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Web page (HTML)
+key: web-page-html
+---
+
+Placeholder for definition which is part of this PR:
```

Comment:
Sorry can we not have placeholder definitions. Perhaps the definition can be a separate PR to expedite definitions getting merged earlier, to allow for them to be referred in rules.

Reply:
@jkodu: Having separate PR for definition has been ruled out by @WilcoFiers: https://github.com/act-rules/act-rules.github.io/pull/868#pullrequestreview-283508565
(at the same time, placeholder has also been ruled out by @WilcoFiers: https://github.com/act-rules/act-rules.github.io/pull/973#discussion_r338995324 and given that I'm not a huge fan of duplicating the def (high risk of drift), I'm not sure what to do…)

My idea with placeholder is to get feedback on the rest and not put the PR into Final Call without onboarding the actual definition into it. Thus, there will at the very least be the 2 weeks Final Call period to catch errors in it.